ballerina/Module.md

```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-This module provides the functionality that is required to access and manipulate data stored in any type of relational database,
+This package provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

Comment:
```suggestion
This module provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

Reply:
shall we revert the change in module.md

Reply:
Fixed in [e697738](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/347/commits/e697738db73e0796ce31139cbc8ac128975d0716)